spec/controllers/admin/posts_controller_spec.rb

```diff
@@ -60,23 +58,20 @@
   end
 
   describe 'PUT update' do
-
-    let(:post) { create :post }
-
     describe 'with valid params' do
-
       it 'updates the requested admin_post' do
-        put :update, id: post.to_param, post: valid_attributes
-        post.reload
+        put :update, id: article.to_param, post: { title: 'Updated' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.